ui/src/common/components/CitationSummaryGraph/CitationSummaryGraph.jsx

```diff
@@ -22,8 +22,8 @@ import styleVariables from '../../../styleVariables';
 import { shallowEqual, abbreviateNumber } from '../../utils';
 import { browser } from '../../browser';
 
-const BAR_WIDTH = 0.5;
-const GRAPH_MARGIN = { left: 40, right: 20, top: 10, bottom: 40 };
+const BAR_WIDTH = 0.75;
+const GRAPH_MARGIN = { left: 42, right: 10, top: 30, bottom: 40 };
```

Comment:
I understand the `top` maybe, but why others?

Reply:
On top I increased it since I moved the y-axis label to the top of the graph (check screenshots). Left is because the numbers in the axis were cut in some screen sizes. On the right we don't need that much margin since I moved the label as well to make it more readable.

Reply:
ah ok, saw that.